e2e/utilities/longhorn-install.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+
+set -x
+
+source ../pipelines/utilities/longhorn_ui.sh
+source ../pipelines/utilities/create_longhorn_namespace.sh
+source ../pipelines/utilities/install_backupstores.sh
+source ../pipelines/utilities/longhorn_status.sh
+source ../pipelines/utilities/longhorn_helm_chart.sh
+source ../pipelines/utilities/create_aws_secret.sh
+source ../pipelines/utilities/longhorn_manifest.sh
+
+# create and clean tmpdir
+TMPDIR="/tmp/longhorn"
+mkdir -p ${TMPDIR}
+rm -rf "${TMPDIR}/"
+
+LONGHORN_NAMESPACE="longhorn-system"
+
+install_longhorn_by_chart(){
```

Comment:
Can we reuse functions in `longhorn_helm_chart.sh` here?

Reply:
@yangchiu, Updated!